Description: Use python3 in the program, template, and setup.py
Author: Martin Pitt <martin.pitt@ubuntu.com>
Forwarded: Not yet, waiting on https://gitorious.org/vitorbaptista/pyppd/merge_requests/1

Index: pyppd/bin/pyppd
===================================================================
--- pyppd.orig/bin/pyppd	2012-06-04 15:25:40.539885921 +0200
+++ pyppd/bin/pyppd	2012-06-04 15:26:25.891888117 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from pyppd import runner
 try:
Index: pyppd/pyppd/pyppd-ppdfile.in
===================================================================
--- pyppd.orig/pyppd/pyppd-ppdfile.in	2012-06-04 15:25:40.559885923 +0200
+++ pyppd/pyppd/pyppd-ppdfile.in	2012-06-04 15:26:29.347888283 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # compressor.py
 @compressor@
Index: pyppd/setup.py
===================================================================
--- pyppd.orig/setup.py	2012-06-04 15:25:40.559885923 +0200
+++ pyppd/setup.py	2012-06-04 15:26:27.755888207 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from distutils.core import setup
 from distutils.command.sdist import sdist as _sdist
